ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-commands.sh

```diff
@@ -23,16 +23,33 @@ fi
 
 echo "************ telco5g gather-pao commands ************"
 
-pao_mg_tag="4.11" # pao must-gather does not have 'latest' tag - setting 4.11 as a workaround for now.
-PULL_BASE_REF=${PULL_BASE_REF:-"master"}
-if [ ${PULL_BASE_REF} != "master" ]
-then
-        pao_mg_tag=${PULL_BASE_REF##release-}
+if [[ -n "${E2E_TESTS_CONFIG:-}" ]]; then
+    readarray -t config <<< "${E2E_TESTS_CONFIG}"
+    for var in "${config[@]}"; do
+        if [[ ! -z "${var}" ]]; then
+            if [[ "${var}" == *"CNF_BRANCH"* ]]; then
+                CNF_BRANCH="$(echo "${var}" | cut -d'=' -f2)"
+            fi
+        fi
+    done
+fi
+
+echo "Running for CNF_BRANCH=${CNF_BRANCH}"
+if [[ "$CNF_BRANCH" == *"4.11"* ]]; then
+    pao_mg_tag="4.11"
+fi
+if [[ "$CNF_BRANCH" == *"4.12"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
+    pao_mg_tag="4.12"
+fi
+if [[ "$CNF_BRANCH" == *"4.13"* ]]; then
+    pao_mg_tag="4.12"
 fi
 
-echo "Running pao-must-gather ${pao_mg_tag}-snapshot..."
+echo "Running PAO must-gather with tag pao_mg_tag=${pao_mg_tag}"
 mkdir -p ${ARTIFACT_DIR}/pao-must-gather
-oc adm must-gather --image=quay.io/openshift-kni/performance-addon-operator-must-gather:${pao_mg_tag}-snapshot --dest-dir=${ARTIFACT_DIR}/pao-must-gather
+echo "OC client version from the container:"
+/usr/local/bin/oc version
+/usr/local/bin/oc adm must-gather --image=quay.io/openshift-kni/performance-addon-operator-must-gather:${pao_mg_tag}-snapshot --dest-dir=${ARTIFACT_DIR}/pao-must-gather
```

Comment:
this is more strictly calling `/usr/local/bin/oc` vs the previous that was using whatever on $PATH, which allow more flexibility.

is there a reason behind this change?

Reply:
can we have that reason in the commit msg?

Reply:
Yes, seems like `/usr/local/bin/` is not in PATH. We use completely different image - `telco-bastion`